internal/vsp/feepayment.go

```diff
@@ -788,21 +735,21 @@ func (fp *feePayment) reconcilePayment() error {
 	fp.mu.Lock()
 	feeHash := fp.feeHash
 	fp.mu.Unlock()
-	var apiErr *BadRequestError
+	var apiErr types.ErrorResponse
 	if errors.As(err, &apiErr) {
 		switch apiErr.Code {
```

Comment:
Any reason why ErrorResponse.Code isn't type ErrorCode?

Without changing that, could make this a little simpler by putting the type conversion in this switch statement, rather than every case statement.

Reply:
Good call. Updated in #decred/vspd/374. Once thats merged I will update this PR.